nx/test/nx/lin_alg_test.exs

```diff
@@ -159,10 +159,36 @@ defmodule Nx.LinAlgTest do
       assert_all_close(Nx.dot(q, r), t, atol: 1.0e-10)
     end
 
+    test "works with complex matrix" do
+      import Nx, only: :sigils
```

Comment:
is it ok if I import this at the top of the module?

Reply:
Totally!